config/announcer/openSUSE:Factory:ARM.yml

```diff
@@ -0,0 +1,31 @@
+sender: Guillaume Gardet <guillaume.gardet@arm.com>
+bodytemplate: '
+
+  Please note that this mail was generated by a script.
+
+  The described changes are computed based on the aarch64 DVD.
+
+  The full online repo contains too many changes to be listed here.
+
+
+  Please check the known defects of this snapshot before upgrading:
+
+  https://openqa.opensuse.org/tests/overview?distri=opensuse&groupid=3&version=Tumbleweed&build={version}
```

Comment:
I have the feeling that the openqa url is wrong

Reply:
No, it is fine, just try: https://openqa.opensuse.org/tests/overview?distri=opensuse&groupid=3&version=Tumbleweed&build=20191109

Reply:
groupid `1` is for `x86_64`, `3` is for `aarch64`